apps/client/src/features/viewers/studio/StudioClock.scss

```diff
@@ -1,8 +1,24 @@
 @use '../../../theme/viewerDefs' as *;
 
 @font-face {
-  font-family: "digital-clock";
-  src: local('digital-7'), url('./../../../assets/fonts/digital-7.monoitalic.ttf') format('truetype') ;
+  font-family: 'seven-seg';
+  src:
+    local('DSEG7'),
+    url('./../../../assets/fonts/DSEG7ClassicMini-Regular.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'fourteen-seg';
+  src:
+    local('DSEG14'),
+    url('./../../../assets/fonts/DSEG14ClassicMini-Regular.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'mono';
+  src:
+    local('MONO'),
+    url('./../../../assets/fonts/Hack-Regular.ttf') format('truetype');
```

Comment:
question: could we use a default monospace font to avoid one more import? I dont know in terms of style if this one is significantly better

Reply:
yeah.. i dont like the default monospace that much so i'm trying to finde one that fits better with the lowtech look